scene/resources/curve.cpp

```diff
@@ -479,6 +479,9 @@ void Curve::set_bake_resolution(int p_resolution) {
 }
 
 real_t Curve::sample_baked(real_t p_offset) const {
+	// Make sure that p_offset is finite.
```

Comment:
IMO the comment is unnecessary, the code is self-explanatory.
At least there is no reason to repeat it in every method.